src/net_processing.cpp

```diff
@@ -1749,11 +1749,7 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
             }
         }
 
-        int minVersion = MIN_PEER_PROTO_VERSION;
-	    // DeepOnion: The code base will fork two weeks before segwit activates, until that point we will allow older versions to connect.
-        if((chainparams.GetConsensus().vDeployments[Consensus::DEPLOYMENT_SEGWIT].nStartTime - 1209600) > GetAdjustedTime()) {
-            minVersion = GETHEADERS_VERSION; // The previous MIN_PEER_PROTO_VERSION.
-        }
+        int minVersion = GETHEADERS_VERSION;
```

Comment:
This needs to stay at MIN_PEER_PROTO_VERSION (which is currently the SEGWIT protocol version), otherwise old wallets will try to connect.

Previous logic was minversion = GETHEADERS_VERSION if segwit fork is more than two weeks in the future.